Scripts/script-IncapScheduleTask.yml

```diff
@@ -0,0 +1,39 @@
+commonfields:
```

Comment:
I forgot to add the description.. :\
This script is not generic. it's specifically runs the white-list compliance script periodically.
Maybe it's name is misleading